lib/propertybase_id/generator.rb

```diff
@@ -32,6 +24,11 @@ def host_id
       @_host_id ||= Digest::SHA1.hexdigest(Socket.gethostname).to_i(16) % PropertybaseId.max_value(2)
     end
 
+    def random_32
+      # signed 32bit integer max range
+      SecureRandom::random_number(2147483648)
```

Comment:
http://ruby-doc.org/stdlib-2.2.2/libdoc/securerandom/rdoc/SecureRandom.html#method-c-random_number

hm they want a number here and there is no `to_i` conversion of a string to base_36 directly. or is there?